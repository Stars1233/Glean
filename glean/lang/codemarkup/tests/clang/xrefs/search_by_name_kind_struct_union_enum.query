query: search.code.SearchByNameKindAndLanguage { name = "E" | "U" | "S", language = Cpp, kinds = { just = Enum_ | Struct | Union } }
perf: true
transform: [gensort, []]
