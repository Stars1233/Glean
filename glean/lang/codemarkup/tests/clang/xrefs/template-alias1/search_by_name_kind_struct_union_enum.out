[
  "@generated",
  {
    "key": {
      "name": "S",
      "language": 0,
      "kinds": 24,
      "entity": {
        "cxx": {
          "defn": {
            "record_": {
              "key": {
                "declaration": {
                  "key": {
                    "name": {
                      "key": {
                        "name": { "key": "S" },
                        "scope": { "global_": { } }
                      }
                    },
                    "kind": { "struct_": { } },
                    "source": {
                      "file": { "key": "main.cpp" },
                      "lineBegin": 10,
                      "columnBegin": 1,
                      "lineEnd": 10,
                      "columnEnd": 11
                    }
                  }
                },
                "bases": [ ],
                "members": { "key": [ ] }
              }
            }
          }
        }
      }
    }
  }
]